.changeset/grumpy-scissors-watch.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
```

Comment:
is this a patch when it's almost certainly a breaking change for existing configs? ig we're saying it's a bugfix?

Reply:
The PR change might be a potential breaking change for some people, but I believe we'd like to avoid as frequent major releases as possible. Because we plan to have various features and breaking changes on the next major version, so the merge of this PR would be blocked until the next major version would be prepared.

For people who would like to keep the current behavior, it might be more kind if there would be a code example like this:

```md
`overrides.extends` is fixed to make consistency with the `overrides.plugins` behavior and be merged not replaced.
If you would like to keep the previous behavior, you would need to change your config as below:
```

```diff
{
- "extends": ["config-a"]
  "overrides": [
    {
      "rules": ["*.module.css"],
      "extends": ["config-b"]
    },
+   {
+     "rules": ["*.css"],
+     "extends": ["config-a"]
+   },
  ]
}
```